.travis.yml

```diff
@@ -1,30 +1,46 @@
 language: python
```

Comment:
`travis setup releases` as suggested on the [Travis docs](https://docs.travis-ci.com/user/deployment/releases/)
is responsible for the whitespace changes.